src/component/CreateEventForm.js

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { createEvent } from '../redux/event/event';
+import '../css/CreateEventForm.css';
+
+const CreateEventForm = () => {
+  const dispatch = useDispatch();
+  const createEventAction = bindActionCreators(createEvent, dispatch);
+
+  const [eventName, setEventName] = useState('');
+  const [eventDescription, setEventDescription] = useState('');
+  const [eventImage, setEventImage] = useState('');
+
+  return (
+    <div className="row bg-image-sign-up mx-0 d-flex flex-column justify-content-center align-items-center">
+      <div className="col-sm-6">
+        <h2 className="light-font">Create new event</h2>
+        <Form
+          onSubmit={async (e) => {
+            e.preventDefault();
+            const formData = new FormData();
+            formData.append('name', eventName);
+            formData.append('description', eventDescription);
+            formData.append('image', eventImage);
+            await createEventAction(formData);
+            setEventName('');
+            setEventDescription('');
+            e.target.reset();
+          }}
```

Comment:
- [ ] This onSubmit code could be transported to a separate function outside of the `return` statement of the component, what do you think?